0001-typedb/01-start/04-iam-schema.md

```diff
@@ -0,0 +1,298 @@
+---
```

Comment:
In this file particularly, I think we should be sure that every time we refer to some concept in TypeQL or code that it is surrounded by backticks. We get relaxed with this at the bottom.